WordPress/Classes/Networking/ReaderPostServiceRemote.m

```diff
@@ -443,10 +445,8 @@ - (NSString *)sortDateFromPostDictionary:(NSDictionary *)dict
  */
 - (NSString *)featuredImageFromPostDictionary:(NSDictionary *)dict
 {
-    NSString *featuredImage = @"";
+    NSString *featuredImage = [NSString string];
```

Comment:
[This post](http://stackoverflow.com/questions/8815980/versus-nsstring-string) is interesting, although I'm not sure it's relevant to picking either.